### PR TITLE
Refresh Button

### PR DESCRIPTION
**Pull request by JRosenfeldIntern**
Added button to reload xml when in the field mapping pane

@MikeMillerGIS @chris-fox

### CODE REVIEW

**Comment by chris-fox** [comment]
@JRosenfeldIntern, I would recommend you use the code below to add the button to the UI. In the change below I am putting the listbox and the button inside a DockPanel in the first row of the grid.

        <Grid.RowDefinitions>
            <RowDefinition Height="Auto" />
            <RowDefinition Height="8" />
            <RowDefinition Height="*" />
        </Grid.RowDefinitions>
        <DockPanel Grid.Row="0" LastChildFill="True">
            <Button DockPanel.Dock="Right" x:Name="Refresh_Button" VerticalAlignment="Center" Style="{StaticResource Button16}" Click="MethodPanelrefreshXML_Click" ToolTip="Refresh the Source-Target Configuration File">
                <Image Source="pack://application:,,,/ArcGIS.Desktop.Resources;component/Images/GenericRefresh16.png"></Image>
            </Button>
            <ListBox DockPanel.Dock="Left" x:Name="MainTabs" Style="{DynamicResource PanelIndicator}" SelectedIndex="0" >
                <ListBox.Items>
                    <frameworkControls:PanelHeader Text="File"/>
                    <frameworkControls:PanelHeader Text="Fields"/>
                </ListBox.Items>
            </ListBox>
        </DockPanel>

In addition, rather than storing the image with the solution I am referencing it from the application resources. This is a trick they don't show in the SDK, but if there is any icon you like in the application you can get it's reference using Snoop. This will ensure you get the correct light and dark theme images.

I noticed that you added 2 new selection changed event handlers to the Dockpane1.xaml.cs that are not used. You should be able to remove these.

**Comment by JRosenfeldIntern** [comment]
Thanks @chris-fox I went ahead and implemented those changes and sent another pull request.

File: Shared/Dockpane1.xaml [hunk 1]
```diff
@@ -75,13 +75,18 @@
...
+                <Image Source="pack://application:,,,/ArcGIS.Desktop.Resources;component/Images/GenericRefresh16.png"></Image>
+            </Button>
+            <ListBox DockPanel.Dock="Left" x:Name="MainTabs" Style="{DynamicResource PanelIndicator}" SelectedIndex="0" >
+                <ListBox.Items>
+                    <frameworkControls:PanelHeader Text="File"/>
+                    <frameworkControls:PanelHeader Text="Fields"/>
+                </ListBox.Items>
+            </ListBox>
+        </DockPanel>
+        <Grid Grid.Row="3" x:Name="FileGrid" ScrollViewer.HorizontalScrollBarVisibility="Disabled" ScrollViewer.VerticalScrollBarVisibility="Auto">
```
Comment (chris-fox) [inline Shared/Dockpane1.xaml]: This grid and the FieldsGrid should remain on Grid.Row="2" since we didn't add a new row.

**Comment by JRosenfeldIntern** [comment]
Fixed the grid row numbers. That was likely due to my original solution having the button as an independent row instead of in the top right.

**Pull request merged**